libraries/AP_OSD/AP_OSD.h

```diff
@@ -162,10 +169,20 @@ class AP_OSD {
     AP_Int8 rc_channel;
     AP_Int8 sw_method;
 
+    AP_Int8 v_offset;
+    AP_Int8 h_offset;
+
+    AP_Int8 warn_rssi;
```

Comment:
I think the warn_rssi better be a AP_Int16, as levels over 127 may be wanted

Reply:
In osd rssi is shown in %, so 99 is the maximum level